tests/test_plugins_os_unix_auth.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime
+from os import stat
+from io import BytesIO
+from zoneinfo import ZoneInfo
+
+from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.plugins.os.unix.log.auth import AuthLogRecord, AuthPlugin
+
+from flow.record.fieldtypes import datetime as dt
+
+from ._utils import absolute_path
+
+# NOTE: stat info of files will be different depending
+# on when the project was cloned by git. That's why we
+# derive the st_ctime of all test data files using os.stat.
+
+
+def test_auth_plugin(target_unix, fs_unix: VirtualFilesystem):
+    fs_unix.map_file_fh("/etc/timezone", BytesIO("Europe/Amsterdam".encode()))
+
+    data_path = "data/unix/logs/auth/auth.log"
+    data_file = absolute_path(data_path)
+    fs_unix.map_file("var/log/auth.log", data_file)
+
+    year = datetime.fromtimestamp(stat(absolute_path(data_path)).st_ctime).year
```

Comment:
Consider mocking the stat result. See https://github.com/fox-it/dissect.target/blob/d828f9498f2c81c38b520702aca8ef5df9172e6e/tests/test_helpers_utils.py#L47 for an example. If you're unsure on how to best approach this, I can do that change myself.

Reply:
Could you implement the mocked stat? Implemented the other suggestions.

Reply:
Added the mocked stat.